packages/opencanary/data_stream/events/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,762 @@
+      tag: remove_opencanary_logdata_ServerName
+      if: ctx.log.logger == 'LOG_MSSQL_LOGIN_SQLAUTH'
+      field: opencanary.logdata.ServerName
+      ignore_missing: true
+      ignore_failure: true
+
+  # Redis Events
+  - rename:
+      description: Rename Redis event opencanary.logdata.ARGS to opencanary.redis.args
+      tag: rename_opencanary_logdata_ARGS
+      if: ctx.log.logger == 'LOG_REDIS_COMMAND'
+      field: opencanary.logdata.ARGS
+      target_field: opencanary.redis.args
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename Redis event opencanary.logdata.CMD to opencanary.redis.command
+      tag: rename_opencanary_logdata_CMD
+      if: ctx.log.logger == 'LOG_REDIS_COMMAND'
+      field: opencanary.logdata.CMD
+      target_field: opencanary.redis.command
+      ignore_missing: true
+      ignore_failure: true
+
+  # SMB Events
+  - rename:
+      description: Rename opencanary.logdata.AUDITACTION to event.action
+      tag: rename_opencanary_logdata_AUDITACTION
+      field: opencanary.logdata.AUDITACTION
+      target_field: event.action
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename opencanary.logdata.FILENAME to opencanary.smb.filename
+      tag: rename_opencanary_logdata_FILENAME
+      field: opencanary.logdata.FILENAME
+      target_field: opencanary.smb.filename
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename opencanary.logdata.SHARENAME to opencanary.smb.share_name
+      tag: rename_opencanary_logdata_SHARENAME
+      field: opencanary.logdata.SHARENAME
+      target_field: opencanary.smb.share_name
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename opencanary.logdata.SMBARCH to opencanary.smb.smb_arch
+      tag: rename_opencanary_logdata_SMBARCH
+      field: opencanary.logdata.SMBARCH
+      target_field: opencanary.smb.smb_arch
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename opencanary.logdata.SMBVER to opencanary.smb.smb_version
+      tag: rename_opencanary_logdata_SMBVER
+      field: opencanary.logdata.SMBVER
+      target_field: opencanary.smb.smb_version
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename opencanary.logdata.STATUS to opencanary.smb.status
+      tag: rename_opencanary_logdata_STATUS
+      field: opencanary.logdata.STATUS
+      target_field: opencanary.smb.status
+      ignore_missing: true
+      ignore_failure: true
+
+  # SSH Events
+  - rename:
+      description: Rename SSH event opencanary.logdata.LOCALVERSION to opencanary.ssh.local_version
+      tag: rename_opencanary_logdata_LOCALVERSION
+      if: ctx.log?.logger == 'LOG_SSH_NEW_CONNECTION' || ctx.log?.logger == 'LOG_SSH_LOGIN_ATTEMPT' || ctx.log?.logger == 'LOG_SSH_REMOTE_VERSION_SENT'
```

For these multi disjunctions, it's better to have the initial null check since if it fails, we can leave. In the construction here, if the first one is null, we need to check the second and so on. Also better to use the syntax that I had [in this comment](https://github.com/elastic/integrations/pull/9532#discussion_r1623717137) with `>-` so that diff future noise is reduced.
```suggestion
      if: ctx.log?.logger == 'LOG_SSH_NEW_CONNECTION' || ctx.log?.logger == 'LOG_SSH_LOGIN_ATTEMPT' || ctx.log?.logger == 'LOG_SSH_REMOTE_VERSION_SENT'
```
```suggestion
      if: >-
        ctx.log?.logger == null && (
          ctx.log.logger == 'LOG_SSH_NEW_CONNECTION' ||
          ctx.log.logger == 'LOG_SSH_LOGIN_ATTEMPT' ||
          ctx.log.logger == 'LOG_SSH_REMOTE_VERSION_SENT'
        )
```

---

I think we should be good on those now.